xml/ha_hawk2_history_i.xml

```diff
@@ -317,11 +317,13 @@
   <title>Viewing Transition Details in the History Explorer</title>
   <para>
    For each transition, the cluster saves a copy of the state which it provides
-   as input to the policy engine (PE). The path to this archive is logged. All
+   as input to <systemitem class="daemon">pacemaker-schedulerd</systemitem>. 
+   The path to this archive is logged. All
    <filename>pe-input*</filename> files are generated on the Designated
```

Comment:
`pe-*` files. Depending on the situations, they might be called pe-input*, pe-warn* or pe-error*.

Reply:
Changed. Did not know about this, thanks.